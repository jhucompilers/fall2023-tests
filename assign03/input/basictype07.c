int main(void) {
  // basic type keywords can appear in any order
  int long unsigned iul;
  char signed cs;
  int short unsigned isu;

  iul = 11;
  cs = 12;
  isu = 13;

  return 0;
}
