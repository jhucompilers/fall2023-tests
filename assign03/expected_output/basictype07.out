0|main|function|function () returning int
2|iul|variable|unsigned long
2|cs|variable|char
2|isu|variable|unsigned short
